vueapp/src/components/Game.vue

```diff
@@ -79,4 +83,10 @@ function addGameFromCourtToQueue(court: Court) {
   gameStore.addGameToOnDeckQueue(court.game)
   courtStore.removeGameFromCourt(court)
 }
+
+function completeGame(court: Court, index: number) {
+  gameStore.addGameToOnDeckQueue(court.game)
```

Comment:
Why are we adding the game back to the on-deck queue only to remove it (2 lines later)????  Just remove the game from the court after sending the players back to the waiting queue

Reply:
Because the removeGameAt() function only removes games from the gamesOnDeck array, I could modify the function to work with courts or create a new function.